### PR TITLE
Upgrading sklearn to 1.0.2

### PR DESCRIPTION
**Pull request by NikhilRaverkar**
*Issue #, if available:*

*Description of changes:*

- version update for sklearn. 
- Added integration tests as a part of codebuild
- added unit test coverage as a part of codebuild
- updated integration test model and setup and verified with codebuild

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docker/1.0-1/final/Dockerfile.cpu
Comment (awsbmillare) [inline docker/1.0-1/final/Dockerfile.cpu:57]: can you document why this is needed now?

**Review by awsbmillare — CHANGES_REQUESTED** [review]
Can you address my other question and also create a branch for the previous version before I can approve this? Once we merge this, this will break the other code builds, we want to make sure the old version build process works before merging.

File: docker/1.0-1/final/Dockerfile.cpu
Comment (NikhilRaverkar) [reply docker/1.0-1/final/Dockerfile.cpu]: Reverted the change. Upon additional testing I found out that this was not needed.

**Review by awsbmillare — APPROVED** [review]

**Pull request merged**
